qiita_ware/commands.py

```diff
@@ -70,22 +71,47 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
     ebi_submission.write_xml_file(ebi_submission.generate_submission_xml(),
                                   ebi_submission.submission_xml_fp)
 
-    # other steps
     if send:
-        ebi_submission.send_sequences()
-        study_accession, submission_accession = ebi_submission.send_xml()
-
-        if study_accession is None or submission_accession is None:
+        # step 4: sending sequences
+
+        # Set the ASCP password to the one in the Qiita config, but remember
+        # the old pass so that we can politely reset it
+        old_ascp_pass = environ.get('ASPERA_SCP_PASS', '')
+        environ['ASPERA_SCP_PASS'] = qiita_config.ebi_seq_xfer_pass
+        try:
+            seqs_cmds = ebi_submission.generate_send_sequences_cmd()
```

Comment:
It looks like `seqs_cmd` is only used to assign the `seqs_cmds_moi`. Probably just better to call it `seqs_cmds_moi` and we avoid one variable aliasing?